include/frg/logging.hpp

```diff
@@ -33,7 +33,7 @@ struct stack_buffer_logger {
 		}
 
 		template<typename T>
-		item &operator<< (T object) {
+		item &operator<< (T &&object) {
 			format(object, *this);
```

Comment:
We probably want to use `std::forward` when passing `object` to `format`. Other than that, this LGTM.